src/server/enterprise/testing/cmd_test.go

```diff
@@ -311,6 +311,10 @@ func TestSyncContexts(t *testing.T) {
 		"clusterId", newClusterId,
 		"userAddress", "grpc://pachd.default:700",
 	).Run())
+
+	// make sure that the cluster with id = 'localhost' does not get synched.
```

Comment:
I think the metric we care about for filtering is actually `EnterpriseServer` being true? The idea is that an org has one enterprise server, and that's where we're getting the info from?

Reply:
thanks! that makes sense. I wasn't on the same page for the semantics of that field